chain-signatures/node/src/mesh/mod.rs

```diff
@@ -57,6 +57,7 @@ impl Mesh {
                 stable.insert(participant, info.clone());
             }
         }
+        tracing::info!(?stable, "stable participants");
```

Comment:
This should be placed at call site instead of inside the function otherwise multiple calls to it will spam the logs and we should know where exactly the context of stable is